pages/index.md

```diff
@@ -17,4 +17,6 @@ We will accomplish our mission by:
 * being design-centric, agile, open, and data-driven
 * deploying tools and services early and often
 
+We know, there is a lot of information on this site, which can be intimidating. Federal hiring processes are more complex than most other places, so we want to make sure you have all the information you need. We're happy you're interested in joining the team!
```

Comment:
My thinking here is that this site is great in that it includes a _ton_ of information, but I worry that it's scaring away potential applicants. Tried to make it sound friendly and inviting, and to allay (or confirm?) their fears that "government application processes are scary".

We might also consider distinguishing pages that are "required" reading before they move to the application, vs. those that are there simply there for reference (or at least linked to in context on the application form). Could be as simple as linking to them from a final sentence in this paragraph. My thinking is that they only really _need_ to read the following pages:

* Who we are hiring (maybe?)
* Roles and teams
* How to apply

Thoughts?